service/src/main/java/bio/terra/policy/app/controller/TpsApiController.java

```diff
@@ -46,6 +52,30 @@ public ResponseEntity<Void> deletePao(UUID objectId) {
     return new ResponseEntity<>(HttpStatus.NO_CONTENT);
   }
 
+  @Override
+  public ResponseEntity<ApiErrorReport> validateDatacenterAllowed(
+      UUID objectId, String datacenter, String platform) {
+    Pao pao = paoService.getPao(objectId);
+    if (regionService.isDatacenterAllowedByPao(pao, datacenter, platform)) {
+      return new ResponseEntity<>(new ApiErrorReport(), HttpStatus.NO_CONTENT);
```

Comment:
I think if you remove the content from the openapi, you can make the response entity with just the status.